examples/hello_world/client/android/java/com/google/oak/hello_world/MainActivity.java

```diff
@@ -41,6 +44,7 @@
 
 /** Main activity for the Oak Android "Hello, World" app. */
 public class MainActivity extends Activity {
+  private static final String OAK_LABEL_GRPC_METADATA_KEY = "x-oak-label-bin";
```

Comment:
Maybe add a comment with a link to where this is defined in the Rust Runtime impl.

Reply:
Done.